assert/assertions.go

```diff
@@ -753,11 +753,11 @@ func Len(t TestingT, object interface{}, length int, msgAndArgs ...interface{})
 	}
 	ok, l := getLen(object)
 	if !ok {
-		return Fail(t, fmt.Sprintf("\"%s\" could not be applied builtin len()", object), msgAndArgs...)
+		return Fail(t, fmt.Sprintf(`"%s" could not be applied builtin len()`, spew.Sdump(object)), msgAndArgs...)
```

Comment:
Why not `%q` instead of `"%s"`?

Reply:
first, this is same as before

second, after `spew.Sdump` there is no need to resulting quote string